templates/web/pages/bmviii_apply.tmpl

```diff
@@ -0,0 +1,469 @@
+
+					<div class="field">
+						<label for="why-bm" class="label">
+						    <p class="bmviii-app-font" style="margin-bottom: 0px">
+						        Why do you want to come to BoilerMake? (100 word limit)
+                            </p>
+                        </label>
+						<div class="control">
+							<textarea class="textarea" style="background-color: #F0F0F0" name="why-bm" required>{{ .FormRefill.WhyBM }}</textarea>
+						</div>
+					</div>
+
+                    <div class="field">
+                        <label for="proj-idea" class="label">
+                            <p class="bmviii-app-font" style="margin-bottom: 0px">
+                                Do you already have a project idea or technology that you're interested in learning? (It's okay if you don't) (100 word limit)
+                            </p>
+                        </label>
+                        <div class="control">
+                            <textarea class="textarea" style="background-color: #F0F0F0" name="proj-idea">{{ .FormRefill.ProjIdea }}</textarea>
+                        </div>
+                    </div>
+
+					<div class="field">
+						<label class="checkbox">
+							{{ if .FormRefill.Is18OrOlder }}
+							<input type="checkbox" name="is-18-or-older" checked required>
+							{{ else }}
+							<input type="checkbox" name="is-18-or-older" required>
+							{{ end }}
+							I am 18 years old or older
+						</label>
+					</div>
+
+					<div class="field">
+						<label class="checkbox">
+							{{ if .FormRefill.MLHCodeOfConduct }}
+							<input type="checkbox" name="mlh-code-of-conduct" checked required>
+							{{ else }}
+							<input type="checkbox" name="mlh-code-of-conduct" required>
+							{{ end }}
+							I have read and agree to the <a href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH Code of Conduct</a>
+						</label>
+					</div>
+
+					<div class="field">
+						<label class="checkbox">
+							{{ if .FormRefill.MLHContestAndPrivacy }}
+							<input type="checkbox" name="mlh-contest-and-privacy" checked required>
+							{{ else }}
+							<input type="checkbox" name="mlh-contest-and-privacy" required>
+							{{ end }}
+							I authorize you to share my application/registration information for event administration, ranking, MLH administration, pre- and post-event informational e-mails, and occasional messages about hackathons in-line with the <a href="https://mlh.io/privacy">MLH Privacy Policy</a>. I further agree to the terms of both the <a href="https://github.com/MLH/mlh-policies/blob/master/prize-terms-and-conditions/contest-terms.md">MLH Contest Terms and Conditions</a> and the <a href="https://mlh.io/privacy">MLH Privacy Policy</a>.
+						</label>
+					</div>
+
+          {{ if eq .FormRefill.Decision 0 }}
+					<div class="field has-text-centered">
+						<button class="button is-red" id="submit-app" type="submit">Submit Application</button>
```

Comment:
This button color is different from the color of the button on the dashboard page

Reply:
Ima ask design what they want for the signup/login/dashboard/apply buttons to look like